audbcards/core/datacard.py

```diff
@@ -41,14 +42,19 @@ class Datacard(object):
             a call to :meth:`audbcards.Datacard.player`
             will store an example audio file
             under
-            ``<sphinx_build_dir>/<path>/<db-name>/<media-file-in-db>``
+            ``<sphinx_build_dir>/<path>/<dataset-name>/``
         sphinx_src_dir: source dir of sphinx.
             If not ``None``
             and ``example`` is ``True``,
             a call to :meth:`audbcards.Datacard.player`
-            will store a wavplot of the example audio file
+            will store a wavform plot of the example audio file
```

Comment:
Would it not be `waveform`? But maybe wavform is a specific term I don't know of.

Reply:
Can be resolved.